component-annotations/src/main/java/org/infinispan/jmx/annotations/package-info.java

```diff
@@ -1,6 +1,8 @@
 /**
- * JMX 2.0-like annotations, with no dependencies on JMX 2.0.  A temporary set of annotations until we eventually
- * baseline on Java 7.  These annotations are for internal use, to allow Infinispan components to expose attributes
- * and operations via JMX.
+ * JMX 2.0-like annotations, with no dependencies on JMX 2.0. A temporary set of annotations until we eventually
+ * baseline on Java 7. These annotations are for internal use, to allow Infinispan components to expose attributes and
```

Comment:
Guess this isn't correct anymore 😆

Reply:
I could change it to : "JMX 2.0-like annotations, with no dependencies on JMX 2.0. These annotations are for internal use..."